### PR TITLE
Update built-in-monitoring.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/built-in-monitoring.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ To view the metrics on the Monitoring page, take the following steps:
 
 ## Metrics retention policy
 
-All monitoring data is kept for 2 days.
+The monitoring data is kept for 7 days in Dedicated Tier and for 3 days in Serverless Tier.
```
Comment (ran-huang) [inline tidb-cloud/built-in-monitoring.md:25]:
```suggestion
- For Dedicated Tier, the monitoring data is kept for 7 days.
- For Serverless Tier, the monitoring data is kept for 2 days.
```

**Review by ran-huang — APPROVED** [review]

File: tidb-cloud/built-in-monitoring.md (hunk 1 above)
Comment (shhdgit) [reply tidb-cloud/built-in-monitoring.md:25]: 3 days on server less.

Comment (ran-huang) [reply tidb-cloud/built-in-monitoring.md:25]: Thanks. Updated.

File: tidb-cloud/built-in-monitoring.md [hunk 2]
```diff
@@ -22,7 +22,8 @@ To view the metrics on the Monitoring page, take the following steps:
 
 ## Metrics retention policy
 
-All monitoring data is kept for 2 days.
+- For Dedicated Tier, the monitoring data is kept for 7 days.
+- For Serverless Tier, the monitoring data is kept for 3 days.
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:26]:
```suggestion
- For Dedicated Tier clusters, the monitoring data is kept for 7 days.
- For Serverless Tier clusters, the monitoring data is kept for 3 days.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**